src/storage/QueryEdgePropsProcessor.cpp

```diff
@@ -39,17 +39,9 @@ kvstore::ResultCode QueryEdgePropsProcessor::collectEdgesProps(
     return ret;
 }
 
-void QueryEdgePropsProcessor::addDefaultProps() {
```

Comment:
Why remove the default props?

Reply:
Because fetch edge does not require default properties.

Reply:
If so, how to know which edge the returned props belongs to?